CHANGELOG.md

```diff
@@ -24,6 +24,10 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ### Dependencies
 
+- Updated the base container to latest PyTorch base container which is based on torch 2.0
+- Container now supports CUDA 12, Python 3.10
+- symengine and vtk dependencies updated
```

Comment:
Can we list what version they've been updated to here?

Reply:
Thanks! Added the details to the Changelog